### PR TITLE
Added es6-object-assign polyfill

### PR DESCRIPTION
**Pull request by bennam**
Previewing components doesn't work in IE 11 so added es6-object-assign polyfill.

### CODE REVIEW

File: src/rsg-components/Preview/Preview.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ import ReactDOM from 'react-dom';
 import { transform } from 'buble';
 import PlaygroundError from 'rsg-components/PlaygroundError';
 import Wrapper from 'rsg-components/Wrapper';
+import es6ObjectAssign from 'es6-object-assign';
```
Comment (sapegin) [inline src/rsg-components/Preview/Preview.js]: I think it should be in `index.js` since it‘s a global polyfill.

**Comment by bennam** [comment]
Added polyfill to index.js

**Comment by sapegin** [comment]
Awesome, thanks!

**Comment by nkt** [comment]
@bennam you added polyfill to dev dependencies 😞

**Comment by sapegin** [comment]
@nkt And it wasn’t visible in the diff ;-(

**Comment by nkt** [comment]
@sapegin yep, I noticed this. I guess we should report this to @github

**Pull request merged**